contents/docs/error-tracking/_snippets/hosting-source-maps.mdx

```diff
@@ -0,0 +1,58 @@
+If you don't upload source maps, PostHog will try to fetch them from your minified JavaScript bundles. This means your source maps must be publicly accessible without authentication.
+
+We **recommend** uploading your source maps to PostHog instead for better reliability.
+
+### Local development
+
+Since source maps need to be publicly accessible for PostHog to fetch them, they won't be available when you're testing your app on `localhost`. You can upload your source maps to PostHog using the `posthog-cli` or you can use a proxy service like [ngrok](https://ngrok.com/) to make your local server accessible publicly.
+
+
+### How PostHog fetches source maps
+
+PostHog will try to access source maps in the following order:
+1. PostHog will look for a `SourceMap` and `X-SourceMap` header when fetching your JavaScript bundle.
+2. PostHog will look for a `//# sourceMappingURL=` comment in your JavaScript bundle.
+3. PostHog will try appending a `.map` extension to the URL of your JavaScript bundle.
+
+### Serving source maps with Next.js
```

Comment:
I think we should recommend using Next.js config (https://github.com/PostHog/posthog-js-lite/tree/main/posthog-nextjs-config) here.

Reply:
Maybe we shouldn't even include this section, it is not the recommended way to do things and will probably confuse people.

Reply:
🤔 Yeah. The problem is the misleading tooltip. I'm gonna reword this to be a debug step instead.
<img width="612" alt="Screenshot 2025-06-26 at 2 18 27 PM" src="https://github.com/user-attachments/assets/216adc92-6170-48d3-9d06-daa6518e5553" />

Reply:
I'll get rid of this and add it back if for some reason the tool tip doesn't ship.